Cloud Application Manager/Analytics/CloudApplicationManagerAnalyticsAWSSetup-Manual.md

```diff
@@ -130,6 +131,10 @@ If you prefer the set-up of your environment to be automated, you can find that
                 "kms:ListResourceTags",
                 "lambda:ListFunctions",
                 "lambda:ListTags",
+                "organizations:DescribeOrganization",
+		"organizations:DescribeOrganizationalUnit",
+		"organizations:ListChildren",
+		"organizations:ListRoots",
```

Comment:
please fix indentation on the 3 lines.  Make sure you are using spaces, not tabs